### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Wuzzy2**
What it says on the tin.

### CODE REVIEW

File: locale/atm.de.tr [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# textdomain: atm
+Your account balance: @1 @2=Ihr Kontostand: @1 @2
+Quit=Ende
```
Comment (Niklp09) [inline locale/atm.de.tr:3]: Maybe "Schließen" instead of "Ende"?

**Review by Niklp09 — COMMENTED** [review]
everything else looks good

**Comment by BuckarooBanzay** [comment]
LGTM except for the "Schliessen" thing :+1:

**Comment by Wuzzy2** [comment]
The button is too small for "Schließen".

IMHO this button should be removed anyway because it's standard practice to not have such a button in formspecs. (Esc always closes.)

**Comment by S-S-X** [comment]
> IMHO this button should be removed anyway because it's standard practice to not have such a button in formspecs. (Esc always closes.)

Were there some challenges to close formspecs on android client and that actually make close buttons useful / easier to access than normal / built in way to close formspecs.

I think it might be back button which, depending on device, was not always that simple to access.
Not sure because I do not really use android client but I remember there's been some complaints and talk about something like this.

**Comment by OgelGames** [comment]
Quit buttons are not needed, even on android:
> When a menu or inventory is displayed:
> - Double tap outside menu area: Close menu
> - Press back: Close menu

https://github.com/minetest/minetest/blob/master/doc/android.md

**Comment by S-S-X** [comment]
Yeah, I think it was that as back button wasn't easily / directly available on some devices (maybe this is already fixed in engine) and nobody reads docs to know that double tapping closes formspecs.

**Comment by Wuzzy2** [comment]
Anyway, I won't change it to "Schließen" in this PR because it's too long. Sorry.

**Review by BuckarooBanzay — APPROVED** [review]
IMO: the close button can be removed in a later PR

**Pull request merged**